Week2/homework/index.js

```diff
@@ -0,0 +1,132 @@
+ 'use strict';
+
+{
+  window.onload = () => main(HYF_REPOS_URL);
+  const HYF_REPOS_URL =
+  'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+
+  //creating a function with fetch() method to fetch the resource.
+  function fetchJSON(url){
+    return fetch (url)
+    .then(response => {
+      if (response.status >= 200 && response.status <= 299) {
```

Comment:
with fetch we have a `status.ok` which it functions the same as this line.
```suggestion
      if (response.ok) {
```